postcodes.ts

```diff
@@ -1,24 +1,24 @@
 const base_endpoint = "https://api.postcodes.io/postcodes/";
 
 // Quicker logging helper function
-export function log(msg) {
+export function log(msg: any) {
   console.log(msg);
 }
 
-export async function validatePostcode(postcode) {
+export async function validatePostcode(postcode: string) {
   const rawresponse = await fetch(`${base_endpoint}${postcode}/validate`);
   const response = await rawresponse.json();
   return response["result"];
 }
 
-export async function geocodePostcode(postcode) {
+export async function geocodePostcode(postcode: string) {
   const response = await fetch(
     `https://api.postcodes.io/postcodes/${postcode}`
   );
   return await response.json();
 }
 
-export function getCoordinates(rawJSON) {
+export function getCoordinates(rawJSON: any) {
```

Comment:
We can make a `PostcodeIOResponse` type.
Good news is you don't need everything in there. _just_ the elements you use in the structure they are.

It'll ignore any data you don't use!